Zend/zend_builtin_functions.c

```diff
@@ -1557,27 +1559,77 @@ static void debug_backtrace_get_args(zend_execute_data *call, zval *arg_array) /
 					 */
 					while (i < first_extra_arg) {
 						zend_string *arg_name = call->func->op_array.vars[i];
-						zval *arg = zend_hash_find_ex_ind(call->symbol_table, arg_name, 1);
-						if (arg) {
-							ZVAL_DEREF(arg);
-							Z_TRY_ADDREF_P(arg);
-							ZEND_HASH_FILL_SET(arg);
+						zval arg;
+						zval *old_arg = zend_hash_find_ex_ind(call->symbol_table, arg_name, 1);
+						zend_attribute *attribute = zend_get_parameter_attribute_str(
+							call->func->op_array.attributes,
+							"sensitiveparameter",
+							sizeof("sensitiveparameter") - 1,
+							i
+						);
+
+						if (attribute != NULL) {
+							last_was_sensitive = 1;
 						} else {
-							ZEND_HASH_FILL_SET_NULL();
+							last_was_sensitive = 0;
+							
 						}
+
+						if (old_arg) {
+							ZVAL_DEREF(old_arg);
+							Z_TRY_ADDREF_P(old_arg);
+							arg = *old_arg;
```

Comment:
```suggestion
							ZVAL_COPY_VALUE(&arg, old_arg);
```

nit: Doesn't affect any functionality but this macro is suggested for clearer intent and to only copy relevant fields.